extensions/amp-addthis/validator-amp-addthis.protoascii

```diff
@@ -0,0 +1,73 @@
+#
+# Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-addthis
+  attr_lists: "common-extension-attrs"
+  extension_spec: {
+    allowed_versions: "0.1"
+    allowed_versions: "latest"
+    name: "amp-addthis"
+  }
+  html_format: AMP
+  tag_name: "SCRIPT"
+}
+
+tags: {  # <amp-addthis>
+  amp_layout: {
+    supported_layouts: FILL
+    supported_layouts: FIXED
+    supported_layouts: FIXED_HEIGHT
+    supported_layouts: FLEX_ITEM
+    supported_layouts: NODISPLAY
+    supported_layouts: RESPONSIVE
+  }
+  attr_lists: "extended-amp-global"
+  attrs: {
+    mandatory: true
+    name: "data-pub-id"
+  }
+  attrs: {
+    name: "data-share-description"
```

Comment:
`data-` attributes are always allowed in AMP documents. So this isn't necessary to list unless you want to add another constraint on it, such as what was done for `data-pub-id` with `mandatory`. If there are no other constraints for this or `data-share-title` then I'd suggest removing them.

Reply:
👍